src/viperleed/calc/classes/rparams/rparams.py

```diff
@@ -692,10 +691,26 @@ def getFortranMpiComp(self, comp='auto'):
             self.FORTRAN_COMP_MPI = ['mpiifort -Ofast', '']
             _LOGGER.debug('Using fortran compiler: mpiifort')
         elif found == 'mpifort':
-            self.FORTRAN_COMP_MPI = [
-                'mpifort -Ofast -no-pie -fallow-argument-mismatch',
-                ''
-                ]
+            # check for the mpifort version
+            mpifort_version = None
+            mpifort_call = "mpifort -Ofast"
```

Comment:
We use single quotation marks everywhere else in this function. It would be good to stick to them also here. I think we decided a while back that single quotes would be our standard. The only exception is the triple quotes in docstrings. Portions of the code from Florian that we haven't touched much still have double quotes, but I would not introduce more with new code, and rather slowly migrate double to single ones in parts of the code that we progressively touch.

Reply:
Yeah, this is why I usually use a code formatter for my own stuff, so I don't have to bother doing this (and indent, line breaks etc.) manually...